packages/default/scss/button/_layout.scss

```diff
@@ -55,22 +55,25 @@
         &-icontext {
             overflow: visible; // IE9
 
+            $icon-spacing-before: -$button-padding-x / 4;
+            $icon-spacing-after: $button-padding-x - $icon-spacing-before;
```

Comment:
There is a theme variable called `$icon-spacing`, we could use that one or extend it. Also, according to design, $icon-spacing-after should be 4px in default theme.

We could perhaps switch to direction independent names such as `$icon-spacing-start` / `$icon-spacing-end`.